src/utils/accessibility/accessibility.test.js

```diff
@@ -1,29 +1,53 @@
 import { expect } from 'chai';
-import { accessibleContrast } from './accessibility';
+import { accessibleContrast, fontSizeInPx } from './accessibility';
 
 describe('Accessibility Utils', () => {
   describe('accessibleContrast()', () => {
     describe('AA', () => {
       it('should be 4.5 for normal scale', () => {
-        expect(accessibleContrast('AA', 12, false)).to.equal(4.5);
-        expect(accessibleContrast('AA', 17, false)).to.equal(4.5);
+        expect(accessibleContrast('AA', 12, 'pt', false)).to.equal(4.5);
+        expect(accessibleContrast('AA', 17, 'pt', false)).to.equal(4.5);
+        expect(accessibleContrast('AA', 14, 'px', false)).to.equal(4.5);
+        expect(accessibleContrast('AA', 23, 'px', false)).to.equal(4.5);
       });
 
       it('should be 3 for large scale', () => {
-        expect(accessibleContrast('AA', 14, true)).to.equal(3);
-        expect(accessibleContrast('AA', 18, false)).to.equal(3);
+        expect(accessibleContrast('AA', 14, 'pt', true)).to.equal(3);
+        expect(accessibleContrast('AA', 18, 'pt', false)).to.equal(3);
+        expect(accessibleContrast('AA', 18, 'pt', true)).to.equal(3);
+        expect(accessibleContrast('AA', 24, 'pt', false)).to.equal(3);
       });
     });
 
     describe('AAA', () => {
       it('should be 7 for normal scale', () => {
-        expect(accessibleContrast('AAA', 12, false)).to.equal(7);
-        expect(accessibleContrast('AAA', 17, false)).to.equal(7);
+        expect(accessibleContrast('AAA', 12, 'pt', false)).to.equal(7);
+        expect(accessibleContrast('AAA', 17, 'pt', false)).to.equal(7);
+        expect(accessibleContrast('AAA', 14, 'px', false)).to.equal(7);
+        expect(accessibleContrast('AAA', 23, 'px', false)).to.equal(7);
       });
 
       it('should be 4.5 for large scale', () => {
-        expect(accessibleContrast('AAA', 14, true)).to.equal(4.5);
-        expect(accessibleContrast('AAA', 18, false)).to.equal(4.5);
+        expect(accessibleContrast('AAA', 14, 'pt', true)).to.equal(4.5);
+        expect(accessibleContrast('AAA', 18, 'pt', false)).to.equal(4.5);
+        expect(accessibleContrast('AAA', 18, 'px', true)).to.equal(4.5);
+        expect(accessibleContrast('AAA', 24, 'px', false)).to.equal(4.5);
+      });
+    });
+
+    describe('fontSizeInPx()', () => {
+      describe('Should', () => {
```

Comment:
I'd prefer to avoid this extra `describe` (the less nesting, the easier it is to read the code). Usually, if it's the only `describe` (like in this case), it can be avoided. Just add `should` in `it()`, like in the tests above (e.g. `"should round up"`).